### PR TITLE
add minor handling for status exceptions

### PR DESCRIPTION
**Pull request by AlwxSin**
Related #126

@TheKevJames should I write some test with mocks or is it enough?

### CODE REVIEW

File: auth/gcloud/aio/auth/session.py [hunk 1]
```diff
@@ -55,6 +59,16 @@ async def close(self) -> None:
 if not BUILD_GCLOUD_REST:
     import aiohttp
 
+
+    async def _raise_for_status(resp: aiohttp.ClientResponse):
```
Comment (TheKevJames) [inline auth/gcloud/aio/auth/session.py:63]: Please add `-> None` to the type signature.

Comment (AlwxSin) [reply auth/gcloud/aio/auth/session.py]: Done

**Comment by AlwxSin** [comment]
@TheKevJames sorry about `resp.json()`. Copied wrong piece of code 🤦‍♂️

**Comment by TheKevJames** [comment]
@AlwxSin no problem -- it's my bad code you copied haha, so I should be the one apologizing :P

**Pull request merged**